big_tests/tests/graphql_account_SUITE.erl

```diff
@@ -169,6 +179,12 @@ end_per_testcase(admin_register_user = C, Config) ->
 end_per_testcase(admin_check_plain_password_hash, Config) ->
     mongoose_helper:restore_config(Config),
     escalus:delete_users(Config, escalus:get_users([carol]));
+end_per_testcase(admin_register_user_limit_error = C, Config) ->
+    Domain = domain_helper:domain(),
+    rpc(mim(), mongoose_account_api, unregister_user, [proplists:get_value(user1, Config), Domain]),
+    rpc(mim(), mongoose_account_api, unregister_user, [proplists:get_value(user2, Config), Domain]),
```

Comment:
It might be good to delete all users here to make sure that if the user creation succeeds, all accounts are deleted and the state is clean.